client/httptransport.go

```diff
@@ -33,7 +33,7 @@ func (ht *httpTransport) call(handler server.Handler, request interface{}, respo
 	req, errNewRequest := http.NewRequest(
 		http.MethodPost,
 		ht.endpoint+"/"+string(handler),
-		bytes.NewBuffer(requestBytes),
+		bytes.NewReader(requestBytes),
```

Comment:
Whats the advantage of using a Reader compared to the bytes.Buffer here?

Reply:
Less internal overhead